kafka-admin/src/main/java/io/strimzi/admin/kafka/admin/handlers/TopicListHandler.java

```diff
@@ -0,0 +1,38 @@
+package io.strimzi.admin.kafka.admin.handlers;
+
+import graphql.schema.DataFetchingEnvironment;
+import io.strimzi.admin.kafka.admin.model.Topic;
+import io.vertx.core.Promise;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public class TopicListHandler {
+    private static final List<Topic> sampleTopics = new ArrayList<>();
```

Comment:
constants should be all upper case.

Reply:
Changed it to be uppercase but again, it is a temporary example.

Reply:
I am removing all these hardcoded data in my extension of this PR.